flask_bootstrap/templates/base/table.html

```diff
@@ -79,10 +83,12 @@
     {% for row in data %}
     <tr>
         {% for title in titles %}
-        {% if title[0] == primary_key %}
-            <th scope="row">{{ row[title[0]] }}</th>
+        {% set key = title[0] %}
+        {% set value = row[key] %}
+        {% if key == primary_key %}
+            <th scope="row">{% if value is string %}{% if safe and key in safe %}{{ value|safe }}{% else %}{% if urlize and key in urlize %}{{ render_links(value) }}{% else %}{{ value }}{% endif %}{% endif %}{% else %}{{ value }}{% endif %}</th>
```

Comment:
Refactor this to multiple lines for better readability.

Reply:
Done, please review the lastest commit.